cli/ts/commands/setVerifyingKeys.ts

```diff
@@ -120,16 +120,17 @@ export const setVerifyingKeys = async (
     const ssStateTreeDepth = Number(ssMatch[1]);
     const ssIntStateTreeDepth = Number(ssMatch[2]);
     const ssVoteOptionTreeDepth = Number(ssMatch[3]);
+
     if (
-      stateTreeDepth !== ssStateTreeDepth ||
-      intStateTreeDepth !== ssIntStateTreeDepth ||
-      voteOptionTreeDepth !== ssVoteOptionTreeDepth
+      Number(stateTreeDepth) !== ssStateTreeDepth ||
+      Number(intStateTreeDepth) !== ssIntStateTreeDepth ||
+      Number(voteOptionTreeDepth) !== ssVoteOptionTreeDepth
```

Comment:
Why we need the `Number` here? also same question for L84:89, since I think the input already define them as `number`?

Reply:
@kittybest fixed now